website/docs/r/azure_secret_backend_role.html.md

```diff
@@ -71,6 +73,9 @@ The following arguments are supported:
   Accepts time suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine default TTL time.
 * `max_ttl` – (Optional) Specifies the maximum TTL for service principals generated using this role. Accepts time
   suffixed strings ("1h") or an integer number of seconds. Defaults to the system/engine max TTL time.
+* `sign_in_audience` - (Optional) Specifies the security principal types that are allowed to sign in to the application. 
+  Valid values are: AzureADMyOrg, AzureADMultipleOrgs, AzureADandPersonalMicrosoftAccount, PersonalMicrosoftAccount.
+* `tags` - (Optional) - A comma-separated string of Azure tags to attach to an application.
```

Comment:
```
tags                        = ["team:engineering","environment:development"]
```

Maybe I have it confused, but it seems like based on the example above, a list of tags is supported, rather than a comma-separated string?